Lumos/Public/Invoke-Lumos.ps1

```diff
@@ -173,8 +173,16 @@ Function Invoke-Lumos {
         }
 
         if ($IncludeOfficeProPlus) {
-            $proPlusThemeValue = if ($Lumos -eq 0) { 4 } else { 0 }
-
+            $proPlusThemeValue = if ($Lumos -eq 0) { 
+                4 
+            } else { 
+                if ("HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\O365ProPlusRetail*") {
```

Comment:
I think this will always evaluate to true. Need schanging to `Test-Path "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\O365ProPlusRetail*"`

Reply:
Mm good catch, I definitely meant to have that in there.